code/modules/mob/living/carbon/human/species/station.dm

```diff
@@ -381,8 +387,154 @@
 
 	suicide_messages = list(
 		"is melting into a puddle!",
-		"is turning a dull, brown color and melting into a puddle!",
-		"is ripping out their own core!")
+		"is ripping out their own core!",
+		"is turning a dull, brown color and melting into a puddle!")
+
+	var/list/mob/living/carbon/human/recolor_list = list()
+
+	species_abilities = list(
+		/mob/living/carbon/human/verb/toggle_recolor_verb,
+		/mob/living/carbon/human/proc/regrow_limbs
+		)
+
+/datum/species/slime/handle_life(var/mob/living/carbon/human/H)
+	var/const/color_shift_trigger = 0.1
+	var/const/icon_update_period = 600 // 1 minute
+	var/const/blood_scaling_factor = 5 // Used to adjust how much of an effect the blood has on the rate of color change. Higher is slower.
```

Comment:
I use these to avoid magic numbers in my code - each of these values are used only within this function, though I guess if you want I can move these up to the slime species level, if you so desire.